### PR TITLE
fix spelling errors

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/nvim/po/ca.po [hunk 1]
```diff
@@ -1362,7 +1362,7 @@ msgstr "E156: Falta el nom del senyal"
 
 #: ../ex_cmds.c:5746
 msgid "E612: Too many signs defined"
-msgstr "E612: S'han definit massa senyals"
+msgstr "E612: S'han definite massa senyals"
```
Comment (justinmk) [inline src/nvim/po/ca.po]: this is a non-english translation file.

File: third-party/cmake/BuildLuv.cmake [hunk 2]
```diff
@@ -106,7 +106,7 @@ elseif(MSVC)
     # Same as Unix without fPIC
     "-DCMAKE_C_FLAGS:STRING=${CMAKE_C_COMPILER_ARG1} ${LUV_INCLUDE_FLAGS}"
     # Make sure we use the same generator, otherwise we may
-    # accidentaly end up using different MSVC runtimes
+    # accidently end up using different MSVC runtimes
```
Comment (justinmk) [inline third-party/cmake/BuildLuv.cmake]:
```suggestion
    # accidentally end up using different MSVC runtimes
```

**Comment by hjung4** [comment]
Changes have been made and committed.

File: third-party/cmake/BuildLuv.cmake [hunk 3]
```diff
@@ -106,7 +106,7 @@ elseif(MSVC)
     # Same as Unix without fPIC
     "-DCMAKE_C_FLAGS:STRING=${CMAKE_C_COMPILER_ARG1} ${LUV_INCLUDE_FLAGS}"
     # Make sure we use the same generator, otherwise we may
-    # accidentaly end up using different MSVC runtimes
+    # accidentally end up using different MSVC runtimes
```
Comment (justinmk) [inline third-party/cmake/BuildLuv.cmake]: i was hoping that you would also correct the other cases in this PR which are still wrong

File: third-party/cmake/BuildMsgpack.cmake [hunk 4]
```diff
@@ -68,7 +68,7 @@ elseif(MSVC)
     "-DCMAKE_C_FLAGS:STRING=${CMAKE_C_COMPILER_ARG1}"
     -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
     # Make sure we use the same generator, otherwise we may
-    # accidentaly end up using different MSVC runtimes
+    # accidently end up using different MSVC runtimes
```
Comment (justinmk) [inline third-party/cmake/BuildMsgpack.cmake]: here is an example of another case

**Comment by hjung4** [comment]
The other cases have also been corrected.

**Pull request merged**